libs/perun/components/src/lib/resources-list/resources-list.component.scss

```diff
@@ -2,7 +2,7 @@ table {
   width: 100% !important;
 }
 
-.mat-row {
+.mat-row-height {
   height: auto;
 }
```

Comment:
```suggestion
```
You can completely delete this attribute in the code, it doesnt bring something new